src/domain_bridge/component_manager.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "domain_bridge/component_manager.hpp"
+
+#include <memory>
+#include <string>
+#include <vector>
+
+using namespace std::placeholders;
```

Comment:
I don't think we need to include these headers (or use the `std::placeholders` namespace)
```suggestion
```

Reply:
Removed in 2e2e4bd